woocommerce/woocommerce
# Add Launch Your Store settings section

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #45107 

This PR adds a new setting section for Launch Your Store site visibility under WooCommerce -> Settings -> General.

Things to do -- these items can be updated in a separate PR once the values and requirements are ready.

1. Add `store pages` link.
2. Add `Editor` link
3. Add `Share your site with a private link` feature.

![Screen Shot 2024-03-07 at 1 18 56 PM](https://github.com/woocommerce/woocommerce/assets/4723145/4b27e09e-6257-4170-988f-8e45695c3500)


### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout this branch and enable `launch-your-store` feature flag.
5. Go to `WooCommerce -> Settings`
6. Confirm `Site visibility` section loads.
7. Change values and click on `Save changes` button.
8. Confirm the changes persist.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Add Launch Your Store settings section

#### Comment 

</details>
